securesignals-ima-devapp/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
As mentioned earlier, this file could probably be removed.

Reply:
This file has been deleted.